gitman/commands.py

```diff
@@ -183,6 +183,7 @@ def delete(*, root=None, force=False):
         common.dedent(level=0)
         common.show("Deleting all dependencies...", color='message', log=False)
         common.show()
+        shell.cd(root)
```

Comment:
* The problem :
On windows we cannot be inside the current directory that we want to remove
I did not manage to find a better place.

* The reasonment :
If I found this cd inside uninstall_dependencies, I will dont know where to cd.

* The solution :
I dont know where to place this elsewhere. So, I will let you find a better, who know better your application architecture, when the PR will be merged